src/server/models/cacheable_queries/user.js

```diff
@@ -31,8 +31,6 @@ const userRoleKey = userId =>
 const userAuthKey = authId =>
   `${process.env.CACHE_PREFIX || ""}texterauth-${authId}`;
 
-export const accessHierarchy = ["TEXTER", "SUPERVOLUNTEER", "ADMIN", "OWNER"];
```

Comment:
INFO (positive critique): I love this -- one constant to rule them all and in the darkness bind them!